### ISSUE TITLE
Not working on LAN Fabric Server 1.21

### ISSUE THREAD

**Issue by xRossa** [RakambdaOrg/FallingTree#746]
### Describe the bug

I just updated my server from 1.20.6 to 1.21, and this mod is the only one that didn't work after changing versions. 
(All mods are the same, only updated to 1.21)

It's the same issue with the already Closed report by GuilhermeUFF.

Works fine single player, but it doesn't work server-side. It doesn't even show on ModMenu when **only** in **server-side**.

I've already tried to run it: 
- All 1.21 versions;
- Fabric API + Falling Tree only;
- Fabric API + Falling Tree + Cloth Config only;
- Together among all mods;
   
None worked but single player.

[Only Fabric API + FallingTree + Cloth-config.log](https://github.com/user-attachments/files/16742178/Only.Fabric.API.%2B.FallingTree.%2B.Cloth-config.log)
[All mods together.log](https://github.com/user-attachments/files/16742179/All.mods.together.log)

### Minecraft version

1.21

### Fabric API version

0.102.0

### Mod version

1.21.0.8

### Configuration

```json
I'm dumb, idk how to get it. (default config, probably)
```

**Comment by xRossa** [comment]
I have might find what caused this.

I updated the server downloading (Photo 1) and following the Fabric website tutorial.
![1](https://github.com/user-attachments/assets/8f75be6f-8d00-4cd7-9495-804ff7ac6e41)

The tutorial says:
> NOTE If you are upgrading your existing server, remember to remove the .fabric folder if the installer did not remove that for you! Or you will get classloading errors.

I did it, and used the "_fabric-installer-1.0.1.jar_" to install the new server files onto my already existing server folder. It created 2 jar files + 'libraries' folder (Photo 2) 

![2](https://github.com/user-attachments/assets/1e37aa41-6dec-447f-87a9-5b9195105016)

I double clicked "_fabric-server-launch_" and it created more folders + the .fabric one again, **BUT** it didnt create a must-to-have "_server_" folder inside the .fabric (Photo 3)
![3](https://github.com/user-attachments/assets/c85426cc-5b0d-4999-ad65-b6ce7de17d94)

Somehow the server can run without the "_server_" folder from the "_.fabric_", and it caused to not work the mod.

I found this when I noticed that the jar file "_fabric-server-mc.[minecraft version here]-loader.0.15.11-launcher.1.0.1_" was missing. I got it downloading the "_Executable Server_" (Photo 1), put it in the 1.21 Server folder and run it there, and it created that "_.fabric/server_" folder that was missing.
